app/apps/server/bridges/http.ts

```diff
@@ -78,16 +78,28 @@ export class AppHttpBridge extends HttpBridge {
 				url: info.url,
 				method: info.method,
 				statusCode: response.status,
-				data: await response.json(),
 				headers: Object.fromEntries(response.headers as unknown as any),
 			};
 
-			if (request.hasOwnProperty('encoding')) {
-				if (request.encoding === null) {
-					result.content = Buffer.from(await response.arrayBuffer()).toString();
-				} else {
-					result.content = await response.text();
-				}
+			const body = Buffer.from(await response.arrayBuffer());
+
+			if (request.encoding === null) {
+				/**
+				 * So, here we have a problem. The property is not appropriately typed
+				 * in the Apps-Engine. Thus, even though it was  previously supported to
+				 * return it as a Buffer. Apps ended up needing to use that same type
+				 * assertion you see below.
+				 */
```

Comment:
@sampaiodiego would like your thoughts here on how to best handle the situation

Reply:
Something interesting to add is that changing the type in the Apps-Engine can be a breaking change 😬

Reply:
This fixes my inability to get a binary response from an http request in an app.

Reply:
@cauefcr You should be able to do that in previous versions of Rocket.Chat 🤔

Reply:
I was using a workspace on the develop branch like a dummy. :see_no_evil: